pyeudiw/satosa/backend.py

```diff
@@ -355,7 +351,16 @@ def redirect_endpoint(self, context, *args):
             all_user_claims, _info["issuer"]
         )
         
-        self.db_engine.update_response_object(nonce, state, internal_resp)
+        try:
+            self.db_engine.update_response_object(nonce, state, internal_resp)
+        except Exception as e:
+            return JsonResponse(
```

Comment:
please add the logging call here

Reply:
Resolved in [c69e7db](https://github.com/italia/eudi-wallet-it-python/pull/89/commits/c69e7db13eddb6b3a16fc2e5d182fea4b08cfba8)